docs.html

```diff
@@ -268,11 +339,77 @@ <h3>Copy and Use it ⬇</h3>
                         sandbox="allow-scripts allow-same-origin">
                         </iframe>
                     </article>  
-            
-            </section>
+                </section>
+
+                <section id="list">
+
+                    <h2>Lists</h2>
+                    <div class="componentDiv">
+
+                        <div class="notifDiv">
+                            
+                            <ul class="list listNotif">
+                              <li class="notif ">Notification 1</li>
+                              <li class="notif ">Notification 2</li>
+                              <li class="notif ">Notification 3</li>
+                              <li class="notif ">Notification 4</li>
+                              <li class="notif ">Notification 5</li>
+                              <li class="notif ">Notification 6</li>
+                              <li class="notif ">Notification 7</li>
```

Comment:
I would suggest adding a scroll bar to the stacked list so it looks more like a notification tray.